frontend/packages/dev-console/src/components/topology/topology-types.ts

```diff
@@ -48,7 +48,7 @@ export interface Edge {
   type?: string;
   source: string;
   target: string;
-  data?: { sbr?: K8sResourceKind };
+  data?: {};
```

Comment:
Why was this change needed? Am I missing something about your change that makes `sbr` (which I assume is service binding request) not be needed?

Reply:
Its not needed here because this data part is variable as data can be different for all edge types.

Reply:
🤔 I suppose that's fair. Feels like we are losing something here and not replacing it with a better solution.